contracts/sei-tester/src/contract.rs

```diff
@@ -549,3 +556,24 @@ pub fn query_denoms_from_creator(
 
     Ok(res)
 }
+
+pub fn query_evm_address(
+    deps: Deps<SeiQueryWrapper>,
+    sei_address: String,
+) -> StdResult<EvmAddressResponse> {
+    let valid_addr = deps.api.addr_validate(&sei_address)?;
+    let querier = SeiQuerier::new(&deps.querier);
+    let res = querier.get_evm_address(valid_addr.to_string())?;
+
+    Ok(res)
+}
+
+pub fn query_sei_address(
+    deps: Deps<SeiQueryWrapper>,
+    evm_address: String,
+) -> StdResult<SeiAddressResponse> {
+    let querier = SeiQuerier::new(&deps.querier);
```

Comment:
is there a way to validate the `evm_address` similar to how we validate the `sei_address` above?

Reply:
also should this be moved into the package itself as opposed to being in the tester contract?

Reply:
Discussed offline. TL;DR The library that I have tried `ethaddr` had to be rolledback due to unstable keccak dep. We may move validations down to lib itself in subsequent PRs (and add them all over to keep the code consistent)